src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.vb

```diff
@@ -192,12 +192,10 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
         Private Function IsExeProject() As Boolean
 
-            Dim obj As Object = Nothing
             Dim OutputType As VSLangProj.prjOutputType
 
             Try
-                GetCurrentProperty(VsProjPropId.VBPROJPROPID_OutputType, "OutputType", obj)
-                OutputType = CType(obj, VSLangProj.prjOutputType)
+                OutputType = OutputTypeProperty
```

Comment:
I made a comment about this in one of the other bugs after investigating a similar issue. We're dying a death of a thousand cuts here and working around lil bits and pieces. I totally agree that we should implement ProjectProperties on the project system which would bring us back to full compat and eliminate a lot of these issues.